src/views/Login.vue

```diff
@@ -1,13 +1,44 @@
 <template>
   <div>
     <h1>This is the login page</h1>
+    <label for="email">E-Mail</label>
```

Comment:
Nitpick: das for attribut des label tags verbindet das label mit dem input element über die input id
Siehe hier: https://www.w3schools.com/tags/att_label_for.asp 

Vorschlag: 
<label **for="email"**>E-Mail</label>
<input v-model="email" **id="email"** type="email" />

Reply:
Ah wow, ich hab dacht ds wär ds name-attribute - muchas gracias für die Erleuchtung 😅